lib/views/course_view/downloaded_pinned_courses_view/content_view.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:gocast_mobile/views/course_view/components/courselist_screen.dart';
+
+/// CourseContentScreen
+///
+/// A reusable widget that displays a list of video cards under a specific title.
+/// This widget is designed to be flexible and can be used to represent various
+/// types of course-related content, such as pinned courses or downloaded courses.
+///
+/// Parameters:
+///   [title] - The title of the content section.
+///   [videoCards] - A list of VideoCard widgets to be displayed under the title.
+class CourseContentScreen extends ConsumerWidget {
+  final String title;
+  final List<Widget> videoCards;
```

Comment:
Can it be a different Widget? Or is it almost the same?

Reply:
To clarify: I meant the widgets in `List<Widget> videoCards`

Reply:
Ah got it, yes true in this case it will be a `VideoCard` Widget.
Updated accordingly ✅